docs/REDIS_CACHE.md

```
@@ -0,0 +1,167 @@
+- `POST /api/cache` - Adicionar/Invalidar cache
+- `DELETE /api/cache?key=prefix:identifier` - Deletar item do cache
+
+## Configuração
+
+### Variáveis de Ambiente
+```bash
+# .env.local
+REDIS_URL=redis://localhost:6379
+```
+
+### Para produção com Redis na nuvem:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)